tests/unit/test_changelog_section.py

```
@@ -0,0 +1,80 @@
+"""Tests for scripts/changelog_section.py (release-notes extraction)."""
+
+from __future__ import annotations
+
+import importlib.util
+from pathlib import Path
+
+_SPEC = importlib.util.spec_from_file_location(
+    "changelog_section",
+    Path(__file__).resolve().parents[2] / "scripts" / "changelog_section.py",
+)
+assert _SPEC and _SPEC.loader
```

Ruff (0.15.17)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)